src/mscorlib/Resources/Strings.resx

```diff
@@ -2396,7 +2399,7 @@
     <value>String cannot have zero length.</value>
   </data>
   <data name="Format_UnknowDateTimeWord" xml:space="preserve">
-    <value>The string was not recognized as a valid DateTime. There is an unknown word starting at index {0}.</value>
+    <value>The string '{0}' was not recognized as a valid DateTime. There is an unknown word starting at index '{1}'.</value>
   </data>
   <data name="Format_UTCOutOfRange" xml:space="preserve">
```

Comment:
Can you add the date time into  Format_UTCOutOfRange and Format_OffsetOutOfRange as well?

Reply:
also Format_MissingIncompleteDate seems like it should have it.